config/default_configs/default_perf.yml

```diff
@@ -1,4 +1,5 @@
 moist: "equil"
+apply_limiter: true
```

Comment:
Why do we want to use limiters in the perf jobs?

Reply:
Oh, I just want to keep it the same as before (We probably want to check the performance of the limiter?). But we don't have to.

Reply:
@nefrathenrici @charleskawczynski Do we want this to be true or false in perf?

Reply:
@charleskawczynski will know for sure, but I assume we should keep it the same for consistency.

Reply:
Charlie is out now and he will be all next week. So I guess for now we want to keep it the same, and then when we have some clarity on why it was on for the perf jobs, we can always change it, if needed.

Reply:
Sounds good to me, thanks!